bedrock/base/templates/includes/protocol/footer/footer-refresh.html

```diff
@@ -0,0 +1,134 @@
+        </span>
+        <span class="mzp-c-button-icon-text">{{ ftl('footer-refresh-donate') }}</span>
+      </a>
+      <div class="moz24-footer-language">
+        {% include 'includes/protocol/lang-switcher-refresh.html' %}
+       </div>
+    </div>
+    <nav class="moz24-footer-secondary">
+      <div class="moz24-footer-legal">
+        <p class="moz24-footer-license" rel="license">
+          {% set moco_link = 'href="%s" data-link-type="footer" data-link-text="Mozilla Corporation"'|safe|format(url('mozorg.home')) %}
+          {% set mofo_link = 'href="https://foundation.mozilla.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer" rel="external noopener" data-link-type="footer" data-link-text="Mozilla Foundation"'|safe %}
+          {{ ftl('footer-refresh-visit-mozilla-corporations', moco_link=moco_link, mofo_link=mofo_link) }}<br>
+          {{ ftl('footer-refresh-portions-of-this-content', url=url('foundation.licensing.website-content'), current_year=current_year|string) }}
+        </p>
+        <ul class="moz24-footer-terms">
+          <li><a href="{{ url('privacy.notices.websites') }}" data-link-type="footer" data-link-text="Privacy">{{ ftl('footer-refresh-websites-privacy-notice') }}</a></li>
+          <li>
+            {# Link to /privacy/websites/cookie-settings/ is a legal requirement and should not be removed. It must be present on every page (Issue 14213). #}
+            <a href="{{ url('privacy.cookie-settings') }}" data-link-type="footer" data-link-text="Cookies">{{ ftl('footer-refresh-websites-cookies') }}</a>
+          </li>
+          <li><a href="{{ url('legal.index') }}" data-link-type="footer" data-link-text="Legal">{{ ftl('footer-refresh-websites-legal') }}</a></li>
+          <li><a href="{{ url('mozorg.about.governance.policies.participation') }}" data-link-type="footer" data-link-text="Community Participation Guidelines">{{ ftl('footer-refresh-community-participation-guidelines') }}</a></li>
+          {% if ftl_has_messages('footer-about-this-site') %}
+            <li><a href="{{ url('mozorg.about.this-site') }}" data-link-type="footer" data-link-text="About this site">{{ ftl('footer-refresh-about-this-site') }}</a></li>
+          {% endif %}
+        </ul>
+      </div>
+    </nav>
+    <img class="moz24-footer-logo-image-bottom" src="{{ static('img/logos/mozilla/footer-refresh/mozilla-in-white.svg') }}" alt="mozilla" width="1376" height="285">
```

Comment:
**nit:** suggest adding `loading="lazy"`
```suggestion
    <img loading="lazy" class="moz24-footer-logo-image-bottom" src="{{ static('img/logos/mozilla/footer-refresh/mozilla-in-white.svg') }}" alt="mozilla" width="1376" height="285">
```

Reply:
We can probably also class this as purely decorative and set the alt text as empty `alt=""`